app/views/main_template.scala.html

```diff
@@ -21,29 +21,29 @@
 @this(govuk_wrapper: views.html.govuk_wrapper, sidebar_di: Sidebar, article_di: Article)
 
 @(title: Title,
-  sidebarLinks: Option[Html] = None,
-  contentHeader: Option[Html] = None,
-  bodyClasses: Option[String] = None,
-  mainClass: Option[String] = None,
-  scriptElem: Option[Html] = None,
-  fullWidth: Boolean = false)(mainContent: Html)(implicit request : Request[_], messages: Messages)
+        sidebarLinks: Option[Html] = None,
+        contentHeader: Option[Html] = None,
+        bodyClasses: Option[String] = None,
+        mainClass: Option[String] = None,
+        scriptElem: Option[Html] = None,
+        fullWidth: Boolean = false)(mainContent: Html)(implicit request: Request[_], messages: Messages)
```

Comment:
I'm not sure that this is the expected format. arguments should be align in one line, now title is in one place rest in the different

Reply:
I agree with @PatrykRudnicki